### ISSUE TITLE
Investigate 'cannot determine loading status' errors on CircleCI

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-e2e-tests#403]
Investigate why these errors happen on CircleCI

```
Error
----------------
timeout: cannot determine loading status
from timeout: Timed out receiving message from renderer: -0.006
 (Session info: chrome=54.0.2840.100)
 (Driver info: chromedriver=2.27.440175 (9bc1d90b8bfa4dd181fbbf769a5eb5e575574320),platform=Linux 3.13.0-106-generic x86_64)
```

**Comment by alisterscott** [comment]
Tried adding LANG=”en_US.UTF-8″ environment variable to CircleCI as per [this post](https://bugs.chromium.org/p/chromedriver/issues/detail?id=1552)

**Comment by alisterscott** [comment]
Any ideas on this @hoverduck? We're seeing it more and more, and not sure why.

Hopefully our move to Docker may eliminate this happening on CircleCI as we'll have more control around Chrome and ChromeDriver versions...

**Comment by alisterscott** [comment]
Tried this https://github.com/Automattic/wp-e2e-tests/pull/408 as per StackOverflow suggestion but still happened

**Comment by hoverduck** [comment]
/shrug

I've submitted a ticket to CircleCI to see if they're aware of anything, but I'm not expecting much back since it's a pretty generic problem.

**Comment by hoverduck** [comment]
CircleCI support responded with a link to that same SO post with the `--dns-prefetch-disable` option and said they're not aware of anything else on their end that would be causing it.  He did offer to look closer at an offending build...but again, I don't expect anything to come out of it.

**Comment by alisterscott** [comment]
CircleCI is using the latest Chromedriver (2.27.440175), but a slightly old version of Chrome: 54.0.2840.100.

I tried updating Chrome via a pre-hook but it tells me it's latest ([here](https://circleci.com/gh/Automattic/wp-e2e-tests/7179))

@hoverduck: any ideas?

**Comment by hoverduck** [comment]
CI support just sent this.  I replied that next week we'll be focusing our efforts on figuring this out and will take memory usage into account...but that the tests have always been executing in that timeframe and it doesn't seem unusual.

> I noticed some of your tests are taking a while to finish, with several around 12~16 seconds each, and the longest around 37 seconds.
> 
> This could have an impact on the performance of the browser, particularly if it's using a lot of memory.
> 
> I suggest to check for how much memory the process is using over-time, and possibly if there are any restarts happening with the driver.

**Comment by hoverduck** [comment]
I tried [adding a feature](https://github.com/Automattic/wp-e2e-tests/pull/408/commits/4dcc3e451f44926aedf728c6aff8ff3aa266d549) to run.sh to have it run every spec file in a fresh browser window instead of reusing the same one.  I went back and forth with several CI runs flipping between this feature and the feature to download the latest Chrome stable (which is now v57).  It seemed like I was getting solid green runs when both features were turned on...but then I still saw one timeout on my [last test run](https://circleci.com/gh/Automattic/wp-e2e-tests/7362).

But still, that's fewer timeouts than we have been seeing.  I'm going to merge that in to run over the weekend and see how it does.
